recipe/meta.yaml

```diff
@@ -11,8 +11,9 @@ source:
 
 build:
   number: 1
-  script: NPY_NUM_BUILD_JOBS=1 {{ PYTHON }} -m pip install . -vvv  # [linux and ppc64le]
-  script: {{ PYTHON }} -m pip install . -vvv  # [not (linux and ppc64le)]
```

Comment:
Btw, the old code wrongly used to install the package twice, didn't it?

Reply:
No, one line ran on `linux ppc64le`, and one line ran on `not linux pp64le`, due to the selectors (yes, selectors in Conda-build are magic comments).